openedx_events/content_authoring/data.py

```diff
@@ -227,3 +227,20 @@ class LibraryCollectionData:
     library_key = attr.ib(type=LibraryLocatorV2)
     collection_key = attr.ib(type=str)
     background = attr.ib(type=bool, default=False)
+
+
+@attr.s(frozen=True)
+class LibraryContainerData:
+    """
+    Data related to a library container that has changed.
+
+    Attributes:
+        library_key (LibraryLocatorV2): a key that represents a content library.
+        container_key (str): identifies the container within the library's learning package (e.g.  unit, section)
```

Comment:
[non-blocking] Should this have its own type, like LibraryLocator or CourseKey, instead of being a plain `str`? This question also applies for the collection_key in LibraryCollectionData.

Reply:
I think we can use only the `LibraryContainerLocator` for the Container events. CC @pomegranited

For the Collections, the events were created before we had a `LibraryCollectionLocator`, so we needed the `LibraryLocator` and the `collection_key: str` to reference it. This can be fixed later.

Reply:
@mariajgrimaldi Ya, as noted, making that change is blocked on https://github.com/openedx/opaque-keys/pull/369, but we'd be happy to fix this in a follow-up.

Reply:
@pomegranited: agreed! Thank you for pointing me to https://github.com/openedx/opaque-keys/pull/369